snippets/iron-elements.cson

```diff
@@ -0,0 +1,18 @@
+# Iron components snippets
+
+".text.html":
+  "iron-ajax":
+    "prefix": "iron-ajax"
+    "body": """
+    <iron-ajax
+            id="$1"
+            auto
```

Comment:
Maybe this should be easily erasable since you might not always want this:

    ${2:auto}